modules/store/commerce_store.module

```diff
@@ -61,3 +61,32 @@ function template_preprocess_commerce_store(array &$variables) {
     $variables['content'][$key] = $variables['elements'][$key];
   }
 }
+
+/**
+ * Helper function to generate add store links.
+ *
+ * @param array $options
+ *   Array of options to pass to the route generation.
+ *
+ * @return \Drupal\Core\Link
+ *   The link object to be sent to.
+ */
+function _commerce_store_get_add_link($options = []) {
```

Comment:
Is there a way we could always link to .add_page, and have .add_page intelligently redirect to the single form if there is one store?